docs/concepts/cluster_queue.md

```diff
@@ -17,8 +62,65 @@ Resources in a cluster are typically not homogeneous. Resources could differ in:
 A `ResourceFlavor` is an object that represents these variations and allows
 administrators to associate them with node labels and taints.
 
+A sample ResourceFlavor looks like the following:
+
+```yaml
+apiVersion: kueue.x-k8s.io/v1alpha1
+kind: ResourceFlavor
+metadata:
+  name: spot
+labels:
+  instance-type: spot
+taints:
+- effect: NoSchedule
+  key: spot
+  value: "true"
+```
+
+You can use the `.metadata.name` to reference a flavor from a ClusterQueue.
+
+### ResourceFlavor labels
```

Comment:
we need to mention that nodes with matching labels should exist for the workload to actually schedule.